Support/lib/ruby_utils.rb

```diff
@@ -0,0 +1,55 @@
+module RubyUtils
+  module_function
+
+  # Try to find the Ruby executable `name`. Return an array containing the
+  # command(s) needed to run the executable, or `nil` if the executable could
+  # not be found.
+  #
+  # Typical usage looks like this:
+  #
+  #     Dir.chdir(ENV['TM_PROJECT_DIRECTORY']) if ENV['TM_PROJECT_DIRECTORY']
+  #     executable = RubyUtils.find_executable('rspec')
+  #     if executable
+  #       system(*executable, '--some', '--additional', 'args')
+  #     else
+  #       # Executable not found, so use fallback / display alert / …
+  #     end
+  #
+  # Supports the following cases:
+  #
+  # 1. If an appropriate `TM_*` environment variable is present and points to an
```

Comment:
This is what I see will happen. An environment variable is configured to run a command for a project/simple script that does not have a Gemfile. Later when running the same command for a project that do have a Gemfile with the requested gem, it will not be used because the environment variable i used instead. I think it would be very easy to forget that the environment variable was set, which then would potentially cause a lot of pain to debug because the wrong version of a gem is used.

Also, to use this for both projects with a Gemfile and without one, I would need to constantly activate and disable the environment variable, if I want the gem from the Gemfile to have higher precedence.